pkg/kube/inject/testdata/inject/hello-tproxy.yaml.injected

```diff
@@ -137,7 +137,7 @@ spec:
           readOnlyRootFilesystem: true
           runAsGroup: 1337
```

Comment:
See here: gid 1337 should have the same effect as uid 1337.

Reply:
This is not the common output of tproxy setting but a special arg in the test case